NAMESPACE

```diff
@@ -23,7 +23,7 @@ export(epidist)
 export(epidist_family)
 export(epidist_formula)
 export(epidist_prepare)
-export(epidist_priors)
+export(epidist_prior)
```

Comment:
why the name change?

Reply:
Because `brms` uses `prior` as its argument rather than `priors` (and so I didn't like `prior = priors`)

Reply:
cool that is reasonable. Did you flag this above and I missed it? If not this kind of stuff is super handy to clearly note even if it seems obvious

Reply:
No I missed flagging. Noted it's good to flag this type of thing